internal/executor/instance/persistentworker/isolation/tart/tart.go

```diff
@@ -77,6 +79,20 @@ func (tart *Tart) Run(ctx context.Context, config *runconfig.RunConfig) (err err
 	defer vm.Close()
 
 	// Start the VM (asynchronously)
+	var preCreatedWorkingDir string
+
+	if tart.mountTemporaryWorkingDirectoryFromHost {
+		tmpDir, err := os.MkdirTemp("", "")
```

Comment:
Do we need to clean this thing up when instance finishes?

Reply:
Good catch, see https://github.com/cirruslabs/cirrus-cli/pull/608/commits/9e90f99b263b5494fc71c8e962443c812d2877f3.